README.md

```diff
@@ -105,271 +123,137 @@ Follow these steps to configure your access token and ingest endpoint:
      SIGNALFX_ENDPOINT_URL=https://ingest.<realm>.signalfx.com
     ```
 
-If you're sending data to an OpenTelemetry Collector, you have to specify the full path like this: `http://<otel-collector-host>:9411/api/v2/spans`
+   If you're sending data to an OpenTelemetry Collector, you have to specify
+   the full path like this: `http://<otel-collector-host>:9411/api/v2/spans`
 
-3. (Optional) Globally enable/disable metrics and/or tracing by setting SIGNALFX_TRACING_ENABLED and/or SIGNALFX_METRICS_ENABLED.
+3. (Optional) Globally enable metrics by setting `SIGNALFX_METRICS_ENABLED`:
     ```bash
-    SIGNALFX_TRACING_ENABLED=true  [defaults to true]
     SIGNALFX_METRICS_ENABLED=false [defaults to false]
```

Comment:
Since the text above says "enable metrics" perhaps here we should have `SIGNALFX_METRICS_ENABLED=true [defaults to false]